lib/matplotlib/widgets.py

```diff
@@ -879,8 +879,7 @@ def _click(self, event):
         if self.ignore(event):
             return
         if event.inaxes != self.ax:
-            self.capturekeystrokes = False
-            self.stop_typing()
+             self.stop_typing()
```

Comment:
To be clear, marking the PR as editable allows mpl committers to push changes to your branch--but that takes more than just commenting here.